src/App/App.jsx

```diff
@@ -1,31 +1,18 @@
-import { useState, useEffect } from "react";
-import { Switch, Route } from "react-router-dom";
-import { onAuthStateChanged } from "firebase/auth";
-import { auth } from "../services/firebase";
+import { BrowserRouter, Switch, Route } from "react-router-dom";
 import Header from "../Header";
 import Footer from "../Footer";
 import Home from "../pages/Home";
 import Cards from "../pages/Cards";
 import Login from "../pages/Login";
 
 function App() {
-    const [user, setUser] = useState(null);
-
-    useEffect(() => {
-        onAuthStateChanged(auth, (currentUser) => {
-            setUser(currentUser);
-        });
-    }, []);
-
-    console.log(user);
     return (
-        <>
+        <BrowserRouter>
```

Comment:
arbitrarily moved Router to App.jsx, I imagine index.js could just hold any context providers